app/controllers/api/vms_controller.rb

```diff
@@ -106,6 +106,7 @@ def edit_resource(type, id, data)
         vm.update!(attrs)
         vm.replace_children(children)
         vm.set_parent(parent)
+        vm.save!
```

Comment:
We're doing one query to `update!` [above](https://github.com/ManageIQ/manageiq-api/pull/994/files#diff-7f92a67a59f1578ea3c05b75fc01361b0d9a576c9bc4473bb6b5e81b2aa66cceR106) then another `save!` here

Could we `assign_attributes(attrs)` then `.save!` with one query?